### PR TITLE
Implement test for ethwallet and zbox's sdk in wasm

### PR DESCRIPTION
**Pull request by Rizary**
This is work in progress PR. It should be merged once all the gosdk function ported to wasm and work properly.

### CODE REVIEW

File: test/index.html [hunk 1]
```diff
@@ -158,234 +155,24 @@
...
-  let result = await WebAssembly.instantiateStreaming(fetch("proxy.wasm"), go.importObject)
-  // console.log(result)
-  go.run(result.instance);
-  console.log('This is initializeConfig ',initializeConfig(
-    "0afc093ffb509f059c55478bc1a60351cef7b4e9c008a53a6cc8241ca8617dfe",
-    // "bls0chain", 31082 /*port*/, "http://dev.0chain.net/dns",
-    "bls0chain", 31082 /*port*/, "https://nine.devnet-0chain.net/dns",
-    // "bls0chain", 31082 /*port*/, "http://six.devnet-0chain.net/dns",
-    // "bls0chain", 31082 /*port*/, "https://seven.devnet-0chain.net/dns",
-    10 /*cleanup worker minutes*/));
```
Comment (shaktals) [inline test/index.html:247]: This initialization is outdated, right?

**Comment by shaktals** [comment]
I ran the php server, then loaded the page and saw all functions running. All seem good, except for this requests that failed:
Selection_195

By the way, our first priority for `wasm`is to migrate the file operations. Can you write tests for those after fixing the `invalid signature` issue?

**Comment by Rizary** [comment]
The error might be because of wroing parameter, I should double check again.

Yes, I'm working on fixing invalid signature issue.

**Comment by shaktals** [comment]
Outdated.

**Pull request closed without merging**